src/mongocxx/exception/error_code.hpp

```diff
@@ -25,6 +25,7 @@ enum class error_code : std::int32_t {
 	k_invalid_client_object = 1,
 	k_invalid_collection_object,
 	k_invalid_database_object,
+        k_invalid_parameter,
```

Comment:
Also, can you add a case for `k_invalid_parameter` to `mongocxx_error_category_impl::message()` ?